src/uwtools/drivers/driver.py

```diff
@@ -352,19 +352,21 @@ def _run_via_local_execution(self):
     # Private helper methods
 
     @property
-    def _resources(self) -> dict[str, Any]:
+    def _run_resources(self) -> dict[str, Any]:
```

Comment:
This came up in Group this week: This method was the only one in `Driver` that was not named to explicitly indicate that it is related to component execution. Now it is.